com.sap.cloud.lm.sl.persistence/src/main/java/com/sap/cloud/lm/sl/persistence/services/AbstractFileService.java

```diff
@@ -307,13 +307,10 @@ public Integer execute(Connection connection) throws SQLException {
                         statement = connection.prepareStatement(getQuery(DELETE_FILE_BY_FILE_ID, tableName));
                         for (String space : spaceToFileIds.keySet()) {
                             for (String fileId : spaceToFileIds.get(space)) {
-                                //This is needed because file might be in multiple chunks
-                                for (String fileChunckId : getFileChunkIds(fileId)) {
-                                    LOGGER.info("Execute DELETE FROM " + tableName + " WHERE FILE_ID=" + fileChunckId + "AND SPACE=" + space);
-                                    statement.setString(1, fileChunckId);
+                                    LOGGER.info("Execute DELETE FROM " + tableName + " WHERE FILE_ID=" + fileId + "AND SPACE=" + space);
```

Comment:
Is this Logger needed?
If yes, could we log the statement which will be executed?

Reply:
Yes this logger logs what files will be deleted.
In my opinion this logger is useful because we can track what files has been deleted.
This will be helpful for further bug investigation.

Reply:
That is more than fine. But could the statement object be used in order to get the query which will be executed? Not to create a query using + signs ...

Reply:
Done.